CITATION.cff

```diff
@@ -0,0 +1,45 @@
+cff-version: 1.2.0
+title: "GLASS: Generator for Large Scale Structure"
+message: >-
+  If you use this software, please cite the associated paper instead.
```

Comment:
```suggestion
  If you use GLASS in your research, please remember to cite its paper.
```

Reply:
I feel this wording is a bit confrontational ^

Reply:
Changed the wording! My aim is to 1) make it more clear that any use of GLASS should be cited, even if it's "only that one function to generate shapes", and 2) that for now there's really only the paper to be cited.

Reply:
Yep makes sense. Reads much better now.